src/Features/LightLimitFix.cpp

```diff
@@ -1007,7 +1007,7 @@ bool LightLimitFix::HasShaderDefine(RE::BSShader::Type shaderType)
 		return true;
 	case RE::BSShader::Type::Effect:
 	case RE::BSShader::Type::Water:
-		return !REL::Module::IsVR();
+		return true;
```

Comment:
Maybe move up `case RE::BSShader::Type::Water:` to return true cases above instead? Otherwise we have 2 groups both returning true and also effect returning true for VR.

Like:
```cpp
switch (shaderType) {
	case RE::BSShader::Type::Lighting:
	case RE::BSShader::Type::Grass:
	case RE::BSShader::Type::Water:
		return true;
	case RE::BSShader::Type::Effect:
		return !REL::Module::IsVR();
	default:
		return false;
	}
```